### PR TITLE
Remove deprecated normalize option

### PR DESCRIPTION
**Pull request by hcho3**
The `normalize=True` option has been [marked as deprecated](https://scikit-learn.org/stable/modules/generated/sklearn.linear_model.LinearRegression.html) in the latest scikit-learn. Use pipelines instead.

### CODE REVIEW

**Review by jacobtomlinson — APPROVED** [review]
Looks great thanks!
